### DISCUSSION TITLE
Older VersionS Download ?

### DISCUSSION THREAD

**Discussion by furbies** [aclap-dev/video-downloadhelper#1207] [Help and Q&A]
### Your question

Hi all

I'm trying to resurrect my old Late '08 Macbook Pro, and I've installed FireFox v78.15.0 esr (64-bit) (which I think is the last compatible version for El Capitan ?) but the current version of Video DownloadHelper isn't compatible. Is there a page where I could download the last compatible version so I could have some ability to grab videos/audio. I do understand it won't be as polished as the cirent version, but that's fine...

tia

furbies 

### Addon & CoApp version and details

9.1.0.42

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

In any Firefox window, do `Ctrl+Shift+a`.  In the search field at the top, enter the name of our favorite extension & search on it.  Using `video down` as your search key should be enough.  When you find the page for installing VDH, look for a `Version History` link.  You will find rather a lot of older versions of VDH.  Good luck finding one that works for you.

You will also need a CoApp.  At the top of every page of this forum, there is a link for `aclap-dev`.  Click that.  Look for the CoApp link on that page.  Click that.  There's a `Releases` link on the right side of that page.  Click that.  Good luck finding one that works for you.

**Reply by furbies** [reply]
I'm going to say that GrampaWildWilly's answer is the right one, but does anyone else know where I can find a list that show MacOs version(s), FireFox VideoDownloader versions and Helper App version compatibility ??

I'm on El Capitan 10.11.6, FireFox 78.15.0 esr (64-bit), Video Downloader v8.2.2.8, and currently using v1.3.0 of the helper app

**Reply by GrampaWildWilly** [reply]
I don't believe I've seen such lists.  I would start with the newest version 8 of VDH & methodically work my way backwards.  It's really easy to uninstall & reinstall the browser extension so it wouldn't take you long to do this.  For the CoApp, go into the Wiki, which has a link to it at the top of every page of this forum.  Look at the section for `CoApp Installation`.  There is a section there for Mac & it tells you something about release numbers of Mac.  Again, I would start with the most recent version that allegedly works on your system & then work my way back until I found something that works.  Working your way back would mean methodically going backwards through the archive of CoApps I already pointed out to you.  If you've settled on 8.2.2.8 & 1.3.0 and that works, don't be tempting fate.  Just stick with that.  Be sure you turn off automatic updates for VDH in the Firefox settings.

**Reply by GrampaWildWilly** [reply]
I think you might be able to use CoApp 2.0.19.  Whatever CoApp version it gives you from the link in the Wiki for older Mac versions is the one you should try first.  Don't assume you have to go all the way back to 1.3.0.  If you can use a version 2 of CoApp, you should.  There have been many improvements in the CoApp & if it will run on your system, it is preferable.

**Reply by furbies** [reply]
I did as GrampaWildWilly suggested and tried different versions of the helper app and v1.3.0 is the last version I can use without crashing it crashing on launch :(

**Reply by GrampaWildWilly** [reply]
Oh well.  Such is life.  You've got an old system.  You said it yourself.  You just have to be happy with whatever works.
